tensorflow_serving/model_servers/server.h

```diff
@@ -43,6 +43,7 @@ class Server {
     tensorflow::int32 grpc_port = 8500;
     tensorflow::string grpc_channel_arguments;
     tensorflow::string grpc_socket_path;
+    tensorflow::int32 grpc_max_threads = 200;
```

Comment:
a function of num_cpus might be more maintainable, compared to this hard-coded value.

maybe `4.0 * port::NumSchedulableCPUs()`

?

Reply:
OK

Reply:
Fixed.